pkg/utils/memory_resources.go

```diff
@@ -68,8 +68,8 @@ func GetNumaMemoryResources() (NumaMemoryResources, error) {
 
 		// Get hugepages
 		hugepageBytes, err := getHugepagesBytes(filepath.Join(sysBusNodeBasepath, numaNode, "hugepages"))
-		if err != nil {
-			return nil, err
+		if err != nil && os.IsNotExist(err) {
+			continue
```

Comment:
We still want to return an error is other error cases

Reply:
this may be the best chocies?😅
```
               // Get hugepages
		huagePagePath := filepath.Join(sysBusNodeBasepath, numaNode, "hugepages")
		_, err = os.Stat(huagePagePath)
		if err != nil && os.IsNotExist(err) {
			continue
		}
		hugepageBytes, err := getHugepagesBytes(huagePagePath)
```

Reply:
How about using `else`?

Reply:
Thank for your suggestion,@marquiz,is this ok?

Reply:
Looks legit to me 👍